tutorials/fundamentals/installation.md

```diff
@@ -1,55 +1,106 @@
-# napari installation tutorial
+---
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.0
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
```

Comment:
I've made this document executable for now purely so I can use the Jupyter book dropdown feature, and also because I think in general we want all tutorials to be MyST markdown eventually. 
Once we merge the dropdown accordion feature (issue #279) we can change this dropdown to match that styling. Do you think there are any dangers to keeping it executable until then @melissawm?